documentation/bibliography.bib

```diff
@@ -17,7 +17,7 @@ @article{AlonsoMalloCanoMoreta2006
   keywords     = {Runge–Kutta–Nyström methods;Stability},
   number       = {1-2},
   pages        = {120--131},
-  title        = {Stability of Runge–Kutta–Nyström methods},
+  title        = {Stability of Runge\Endash Kutta\Endash Nyström methods},
```

Comment:
It's hard to believe that these changes are an improvement.  We should strive to keep the source comprehensible.  In particular, I am pretty sure that the next time I reference RKN methods I will forget about the silly `\Endash`.